src/indexer/repository/transfer.ts

```diff
@@ -55,6 +58,13 @@ class TransferRepository {
         },
       },
       timestamp: new Date(decodedLog.timestamp * 1000), // this is only being used by evm service
+      account: {
+        create: {
```

Comment:
This will create a new account for each new deposit, even if the same user is initiating new deposits. We should probably use upsert instead of create

Reply:
Gotcha